provider/validation_helpers.go

```diff
@@ -0,0 +1,13 @@
+package provider
+
+import (
+	"fmt"
+)
+
+func logicKeyValidation(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
Thanks for adding this!  Although I probably should have let you know about the `validation` package which gives you the `StringInSlice` function that could have made this implementation easier.  Take a look at this example: https://github.com/mrparkers/terraform-provider-keycloak/blob/f4770b0a3726a82d0fd0176cba328b4b2dda1389/provider/resource_keycloak_ldap_user_federation.go#L65

Could you change this to use this function?

Reply:
Oh cool! Just replaced that with the validation package. 👍